src/routes/single-page-app.js

```diff
@@ -17,8 +17,8 @@ singlePageAppRouter.get("/*", (req, res) => {
     .map((r) => r.path)
     .includes(req.path);
   if (shouldRedirect) {
-      res.status(301).location(pageRoutes.home.path).send();
-      return;
+    res.status(301).location(pageRoutes.home.path).send();
```

Comment:
Oops, sorry.

Reply:
no problem! the linter in sublime text just does it automatically when i save so i only notice when i see the diff...